kore/src/Kore/Internal/Predicate.hs

```diff
@@ -227,6 +229,14 @@ coerceSort
     -> Predicate variable
 coerceSort sort = fmap (TermLike.fullyOverrideSort sort)
 
+{- | Forget the sort of a 'Predicate'. Used in testing.
+-}
+forgetSort
+    :: (HasCallStack, InternalVariable variable)
+    => Predicate variable
+    -> Predicate variable
+forgetSort = coerceSort Sort.predicateSort
```

Comment:
Please move this to the test suite so that we don't accidentally use it anywhere else.

Reply:
I ended up removing this.